test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -23,12 +23,13 @@ class SolidityCompilationFramework
 	}
 	bytes compileContract(
 		std::string const& _sourceCode,
-		std::string const& _contractName = {}
+		std::string const& _contractName,
+		std::map<std::string, solidity::util::h160> const& _libraryAddresses = {},
+		frontend::OptimiserSettings _optimization = frontend::OptimiserSettings::minimal()
 	);
 protected:
 	frontend::CompilerStack m_compiler;
 	langutil::EVMVersion m_evmVersion;
-	frontend::OptimiserSettings m_optimiserSettings = frontend::OptimiserSettings::none();
```

Comment:
In keeping with the setting meant to be used when "no optimization" is desired. See https://github.com/ethereum/solidity/pull/8400#discussion_r395530092